common/web/types/src/kmx/kmx-plus.ts

```diff
@@ -274,6 +288,24 @@ export class KMXPlusFile extends KMXFile {
 
     // 'bksp' - see 'tran'
 
+    // 'disp'
+    this.COMP_PLUS_DISP_ITEM = new r.Struct({
+      to: r.uint32le,
+      display: r.uint32le,
+    });
+
+    this.COMP_PLUS_DISP = new r.Struct({
+      ident: r.uint32le,
+      size: r.uint32le,
+      count: r.uint32le,
+      baseCharacter: r.uint32le,
+      reserved0: new r.Reserved(r.uint32le), // padding
+      reserved1: new r.Reserved(r.uint32le), // padding
+      reserved2: new r.Reserved(r.uint32le), // padding
+      reserved3: new r.Reserved(r.uint32le), // padding
```

Comment:
Why do we need 16 bytes of padding? We are not padding to 32 bytes are we?

Reply:
See the spec. I added padding in anticipation of other options. Otherwise it would have no reserved area.